ackermann_steering_controller/src/ackermann_steering_controller.cpp

```diff
@@ -62,28 +62,29 @@ bool AckermannSteeringController::update_odometry(const rclcpp::Duration & perio
   }
   else
   {
-    const double rear_right_wheel_value = state_interfaces_[STATE_TRACTION_RIGHT_WHEEL].get_value();
-    const double rear_left_wheel_value = state_interfaces_[STATE_TRACTION_LEFT_WHEEL].get_value();
-    const double front_right_steer_position =
-      state_interfaces_[STATE_STEER_RIGHT_WHEEL].get_value();
-    const double front_left_steer_position = state_interfaces_[STATE_STEER_LEFT_WHEEL].get_value();
+    const double traction_right_wheel_value =
+      state_interfaces_[STATE_TRACTION_RIGHT_WHEEL].get_value();
+    const double traction_left_wheel_value =
+      state_interfaces_[STATE_TRACTION_LEFT_WHEEL].get_value();
+    const double right_steer_position = state_interfaces_[STATE_STEER_RIGHT_WHEEL].get_value();
+    const double left_steer_position = state_interfaces_[STATE_STEER_LEFT_WHEEL].get_value();
```

Comment:
Let's follow traction vars pattern
```suggestion
    const double steer_right_position = state_interfaces_[STATE_STEER_RIGHT_WHEEL].get_value();
    const double steer_left_position = state_interfaces_[STATE_STEER_LEFT_WHEEL].get_value();
```